src/app/modules/item/pages/item-current-situation/item-current-situation.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'alg-item-current-situation',
+  templateUrl: './item-current-situation.component.html',
+  styleUrls: ['./item-current-situation.component.scss']
+})
+export class ItemCurrentSituationComponent implements OnInit {
+
+  viewItems = [
+    {label: 'Chapter view', route: './chapter'},
+    {label: 'Log view', route: './'}
+    ];
```

Comment:
Styling for consistency with the rest of the code:
```suggestion
  viewItems = [
    { label: 'Chapter view', route: './chapter' },
    { label: 'Log view', route: './' }
  ];
```